resources/assets/lib/quick-search/main.tsx

```diff
@@ -103,7 +103,14 @@ const otherModes: ResultMode[] = ['forum_post', 'wiki_page'];
     }
   };
 
-  private onMouseLeave = (event: React.MouseEvent<HTMLInputElement>) => {
+  private onMouseEnter = (event: React.MouseEvent<HTMLDivElement>) => {
+    const section = event.currentTarget.dataset.section as Section;
+    const index = parseInt(event.currentTarget.dataset.index ?? '0', 10);
```

Comment:
kind of moot since these values are set privately, but the function should bail out if they're not defined/valid instead of coalesced 🤔

Reply:
The `_others` don't actually have index...

Reply:
🤔